src/util/compose.js

```diff
@@ -85,6 +85,7 @@ function generateNhostBackendYaml(options) {
           PORT: hasura_backend_plus_port,
           USER_FIELDS: "",
           USER_REGISTRATION_AUTO_ACTIVE: "true",
+          DATABASE_URL: `postgres://${postgres_user}:${postgres_password}@nhost-postgres:5432/postgres`,
```

Comment:
Shouldn't this be nhost_postgres to match the container name ... in the compose file? This is what corresponds to my running of it on Windows 10 under WSL2 .. 

![image](https://user-images.githubusercontent.com/14859781/120084436-171bdd80-c09e-11eb-9551-92fad0f7d47a.png)

Reply:
Both `nhost-postgres` and `nhost_postgres` are valid. `nhost-postgres` is the "node"'s name and `nhost_postgres` is the container name:

![image](https://user-images.githubusercontent.com/331818/120148941-4723c780-c1e9-11eb-82e3-8cbedc509eb9.png)